x-pack/legacy/plugins/epm/public/hooks/use_package_install.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useState, Fragment } from 'react';
+import createContainer from 'constate';
+import { EuiFlexGroup, EuiFlexItem, EuiButton } from '@elastic/eui';
+import { useCore, useLinks } from '.';
+import { installPackage as fetchInstallPackage } from '../data';
```

Comment:
I'm curious about the renaming. I find the double verb of "fetch install" odd, but would like to hear why you prefer it to `installPackage`

Reply:
The only reason I did it is because I already have a fetchInstall function in this function.

Reply:
💡 😄